apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Logging.V2;
+using Google.Cloud.Diagnostics.Common;
+using Google.Protobuf.WellKnownTypes;
+using Google.Api.Gax;
+using Microsoft.Extensions.Logging;
+using System;
+using Google.Api;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    /// <see cref="ILogger"/> for Stackdriver Logging.
+    /// </summary>
+    public class GoogleLogger : ILogger
```

Comment:
I'd suggest sealing this unless there's a reason not to. (I generally like to design for inheritance or prohibit it.)